### DISCUSSION TITLE
how to use ssh setting ?

### DISCUSSION THREAD

**Discussion by timeisgoing** [dbeaver/dbeaver#20729] [Q&A] [answered]
hello，
I want to use this tool in production environment. However, in the production environment, I have to log in to the jumpserver a, and then go to the target host-b before I can log in to the database-c. Can this kind of proxy be set up using ssh?
Thanks

**Comment by E1izabeth** [comment, marked as answer]
Are you looking for this https://dbeaver.com/docs/wiki/SSH-Configuration/#configuring-gateway-host?
